src/sonic-config-engine/tests/common_utils.py

```diff
@@ -40,7 +40,11 @@ def cmp(file1, file2):
             obj1 = json.load(f)
         with open(file2, 'r') as f:
             obj2 = json.load(f)
-        return obj1 == obj2
+        if obj1 == obj2:
+            return true
+        else:
+            os.system("git diff --no-index " + file1 +" "+ file2)
```

Comment:
I would like to make the feature more general, without git dependency.
User can copy the code in a none-git environment to do test. For the feature, it has no dependency on git.

https://stackoverflow.com/questions/28838170/multilevel-json-diff-in-python